resources/install.rb

```diff
@@ -56,4 +58,11 @@ def hab_path
       "/bin/hab"
     end
   end
+
+  def hab_command
+    cmd = [ "bash #{Chef::Config[:file_cache_path]}/hab-install.sh" ]
+    cmd.push("-v #{new_resource.version}") if new_resource.version
+    cmd.push("-c #{new_resource.channel}") if new_resource.channel
+    cmd.join(" ")
```

Comment:
the execute resource takes an array, although you'll have to bust up the -v and -c args into two elements